web/modules/custom/dpl_admin/dpl_admin.links.menu.yml

```diff
@@ -3,8 +3,18 @@ dpl_admin.add_content:
   parent: system.admin_content
   route_name: dpl_admin.add_content
   weight: 5
+dpl_admin.redirect:
+  title: "Redirects"
+  parent: system.admin
+  route_name: redirect.list
+  weight: 10
+dpl_admin.redirect_add:
+  title: "Add redirect"
+  parent: dpl_admin.redirect
+  route_name: redirect.add
+  weight: 1
```

Comment:
I think we need to discuss our strategy for adding items to the menu bar.

It might be me but I think all these top level custom menu entries are getting a bit out of hand.
As I see it they clutter up the menu contents for everyone but editors - and while they are many we have quite a few local administrators, administrators as well. They will have to access the Drupal menu tree anyway. While having direct access it nice these items also represent functionality that I do not think will be used on a day to day basis.